public/scripts.js

```diff
@@ -53,6 +66,62 @@ document.addEventListener('DOMContentLoaded', function() {
     }, 1000);
   }
 
+  function showTalkingHead(author, quote) {
+    var img = new Image();
+    var author = author.toLowerCase();
+
+    img.onload = function() {
+      canvas.style.visibility = 'visible';
+      makeAuthorTalk(this, author, quote);
+    };
+    img.src = 'images/faces/' + author.toLowerCase() + '.png';
+  }
+
+  function muteTalkingHead() {
+    clearInterval(canvasInterval);
+  }
+
+  function hideTalkingHead() {
+    muteTalkingHead();
+    canvas.style.visibility = 'hidden';
```

Comment:
Is hiding and showing the canvas really necessary? Just thinking, since you're clearing it all the time for its content, couldn't you just show an empty canvas? Or wouldn't that necessarily be transparent?